packages/fast-components-styles-msft/src/utilities/color/recipe.ts

```diff
@@ -0,0 +1,113 @@
+import {
+    checkDesignSystemResolver,
+    DesignSystem,
+    DesignSystemResolver,
+} from "../../design-system";
+import { backgroundColor } from "../design-system";
+import { Swatch, SwatchFamily } from "./common";
+import {
+    findClosestSwatchIndex,
+    findSwatchIndex,
+    getSwatch,
+    isDarkMode,
+    Palette,
+    swatchByContrast,
+} from "./palette";
+
+/**
```

Comment:
It's just here for the swatchByContrast call. I could inline it, but it might be harder to read with all the parameters inline. I was considering default implementations of these swatchByContrast methods though, since most of the time we are doing the same thing.

Reply:
In the interest of bundle size I think you should remove - it doesn't provide new utility but adds code.

Reply:
In the interest of bundle size I think you should remove - it doesn't provide new utility but adds code.